scripts/symbiotic

```diff
@@ -56,8 +56,9 @@ where OPTS can be following:
     --timeout=t               Set timeout to t seconds
     --no-slice                Do not slice the code
     --no-prepare              Do not prepare the code
-    --tool=name               Use the tool 'name'. Default is KLEE, other tools that
-                              can be integrated are Ceagle, Skink and SMACK
+    --verifier=name           Use the tool 'name'. Default is KLEE, other tools that
```

Comment:
There was a naming inconsistency: `--tool` here, vs. `--verifier` below - I've changed it to the one used in the code, but maybe you actually wanted `--tool`?

Reply:
Nope, verifier is fine :)